ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -739,33 +856,19 @@ export const RuntimePanel = fp.flow(
                      style={{width: '10rem'}}
                      options={[ComputeType.Standard, ComputeType.Dataproc]}
                      value={selectedCompute || ComputeType.Standard}
-                     onChange={({value}) => setSelectedCompute(value)}
+                     onChange={({value}) => {setSelectedCompute(value); }}
                      />
            {
              selectedCompute === ComputeType.Dataproc &&
-             <DataProcConfigSelector onChange={setSelectedDataprocConfig} dataprocConfig={selectedDataprocConfig} />
+             <DataProcConfigSelector onChange={config => setSelectedDataprocConfig(config)} dataprocConfig={selectedDataprocConfig} />
            }
          </FlexColumn>
        </div>
-       {runtimeExists && runtimeChanged && <FlexRow
-           style={{
-             alignItems: 'center',
-             backgroundColor: colorWithWhiteness(colors.warning, .9),
-             border: `1px solid ${colors.warning}`,
-             borderRadius: '5px',
-             color: colors.dark,
-             marginTop: '.5rem',
-             padding: '.5rem 0px'
-           }}
-       >
-         <ClrIcon
-             style={{color: colors.warning, marginLeft: '.5rem'}}
-             shape={'warning-standard'}
-             size={16}
-             class={'is-solid'}
-         />
-         <div style={{marginLeft: '.5rem'}}>You've made changes that require recreating your environment to take effect.</div>
-       </FlexRow>}
+       {runtimeExists && runtimeChanged &&
+         <WarningMessage>
+            <div>You've made changes that require recreating your environment to take effect.</div>
```

Comment:
This can be a follow-up, but I would have thought that the message here would be different in the update vs delete case.